### DISCUSSION TITLE
给龙猫大大的更新建议

### DISCUSSION THREAD

**Discussion by ZhangHappyHi** [Meloong-Git/PCL#2692] [讨论]

**Comment by WForst-Breeze** [comment]
新闻主页是最亮的信标 @Light-Beacon 做的，不是龙猫，等等作者的回复？

Edit: 我记得新闻主页在 Minecraft 正式版发行的时候会列出这个版本所有的更新内容的，不知道你想表达什么意思🤔

**Comment by Light-Beacon** [comment]
关于新闻主页的问题请在[新闻主页Repo](https://github.com/Light-Beacon/PCL2-NewsHomepage)发Issue或讨论，感谢
